labs/ipfs/how-to/add-pin.mdx

```diff
@@ -0,0 +1,46 @@
+---
+meta:
+  title: How to add a pin to your data and retrieve it
+  description: This page explains how to add a pin to your data and retrieve it using Scaleway IPFS Pinning
+content:
+  h1: How to add a pin to your data and retrieve it
+  paragraph: This page explains how to add a pin to your data and retrieve it using Scaleway IPFS Pinning
+tags: ipfs-pinning ipfs-storage ipfs pinning storage labs web3 volume pin
+dates:
+  validation: 2023-06-27
+  posted: 2023-06-27
+categories:
+  - labs
+---
+
+Scaleway IPFS Pinning allows you to permanently store a copy of your data from the public IPFS network on a Scaleway-owned node, thus providing you with an added layer of performance in the region of your choice. That way, your data remains available and accessible, even when your local machine is offline. 
+
+Pinning a file essentially marks it as important. When data is pinned using Scaleway pinning service, it cannot be removed from our node, thus guaranteeing its availability on the IPFS network at all times.
```

Comment:
It CAN be removed from our nodes !!!

However, from the key step where a data is available and is shared to other IPFS nodes, we cannot interfere and remove data from these external nodes.